cli/coda.ts

```diff
@@ -37,6 +37,11 @@ if (require.main === module) {
           desc: 'Execute the requested command in a virtual machine that mimics the environment Coda uses to execute Packs.',
           default: true,
         },
+        getSchema: {
```

Comment:
Nice.

Although, I wonder if we want to generalize how to invoke the many metadata formulas attached to a sync table. Should the flag instead be something like `--metadataFormula=getSchema`? (Find to handle this separately.) WDYT @ekoleda-codaio ?

Reply:
Ya, the ability to test more of a Pack from the CLI is something I know I've wanted. +1 to making this flexible, but also understand the scope creep aspect.

Reply:
I reverted these changes from this PR. let's make that a separate change to support `--metadataFormula`